db/collab.go

```diff
@@ -13,9 +13,15 @@ import (
 )
 
 // Session describes a collaborative coding environment.
+type Message struct {
+	Author string
+	Type   string
+	Body   string
+}
```

Comment:
To make life easier for concrete message passing later on, this struct should probably be JSON-annotated.